### DISCUSSION TITLE
Vuetify extends wrong namespace from Vue

### DISCUSSION THREAD

**Discussion by Joebayld** [vuetifyjs/vuetify#20428] [General]
Below is a snipped of the index.d.mts file from Vuetify. 

```typescript
declare module '@vue/runtime-core' {
  interface Vuetify {
    defaults: DefaultsInstance
    display: UnwrapNestedRefs<DisplayInstance>
    theme: UnwrapNestedRefs<ThemeInstance>
    icons: IconOptions
    locale: UnwrapNestedRefs<LocaleInstance & RtlInstance>
    date: DateInstance
  }

  export interface ComponentCustomProperties {
    $vuetify: Vuetify
  }
  export interface GlobalComponents extends _GlobalComponents {}
}
```

But according to the Vue documentation, but I believe this should now be just `declare module 'vue'`. Can anyone verify this? Here is a link from the Vue docs. https://vuejs.org/guide/typescript/options-api.html#augmenting-global-properties

If you look at the vue-router releases, they had the same issue and fixed it. I'm having issues where I can't use `{{ $route }}` because any library that uses `declare module '@vue/runtime-core'  essentially causes all other shared namespaces to have issues.  

Here's a link to the release notes of Vue Router talking about it. 
https://github.com/vuejs/router/blob/main/packages/router/CHANGELOG.md#bug-fixes-2

**Comment by KaelWD** [comment]
7c8aeef
